plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildInputs/ItemViewModel.cs

```diff
@@ -0,0 +1,298 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.UniversalReference
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel;
+    using System.Linq;
+
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.Common.EntityMergers;
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.YamlSerialization;
+
+    using Newtonsoft.Json;
+    using YamlDotNet.Serialization;
+
+    [Serializable]
+    public class ItemViewModel : IOverwriteDocumentViewModel
+    {
+        [YamlMember(Alias = Constants.PropertyName.Uid)]
+        [JsonProperty(Constants.PropertyName.Uid)]
+        [MergeOption(MergeOption.MergeKey)]
+        public string Uid { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.CommentId)]
+        [JsonProperty(Constants.PropertyName.CommentId)]
+        public string CommentId { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Id)]
+        [JsonProperty(Constants.PropertyName.Id)]
+        public string Id { get; set; }
+
+        [YamlMember(Alias = "isEii")]
+        [JsonProperty("isEii")]
+        public bool IsExplicitInterfaceImplementation { get; set; }
+
+        [YamlMember(Alias = "isExtensionMethod")]
+        [JsonProperty("isExtensionMethod")]
+        public bool IsExtensionMethod { get; set; }
+
+        [YamlMember(Alias = Constants.PropertyName.Parent)]
+        [JsonProperty(Constants.PropertyName.Parent)]
+        [UniqueIdentityReference]
+        public string Parent { get; set; }
```

Comment:
should be string...

---
In reply to: [112870641](https://github.com/dotnet/docfx/pull/1574#discussion_r112870641) [](ancestors = 112870641,112604924)